examples/terraform/aws-dualstack/credentials_ipv4+ipv6.yaml

```diff
@@ -0,0 +1,5 @@
+cloudConfig: |
```

Comment:
Let's replace `+` in the filename with `_` if possible. I tend to avoid using special symbols in filenames for compatibility reasons.